client/src/app/site/login/components/login-info/login-info.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
+    selector: 'os-login-info',
+    templateUrl: './login-info.component.html',
+    styleUrls: ['./login-info.component.scss']
+})
+export class LoginInfoComponent implements OnInit {
+    /**
+     * triggers what component to show
+     */
+    public component: string;
+
+    /**
+     * Determine the title of the page
+     */
+    public pageTitle: string;
+
+    /**
+     * Imports advanced Router
+     * @param route
+     */
+    public constructor(private route: ActivatedRoute) {}
+
+    /**
+     * Small info wrapper.
+     *
+     * Takes an URL and decides what information to show.
+     */
+    public ngOnInit() {
+        if (this.route.snapshot.url[0]) {
```

Comment:
THis is not really testable. Can you just make two components for the PP and LN each? So having a login-ln-info and login-pp-info and resolve this code with the router.